src/jv.h

```diff
@@ -5,6 +5,20 @@
 #include <stdint.h>
 #include <stdio.h>
 
+#if defined(DISPATCH_FALLTHROUGH)
+# define JQ_FALLTHROUGH DISPATCH_FALLTHROUGH
+#else
+# if defined(__GNUC__)
+#  if __GNUC__ >= 7
```

Comment:
Can we just check `__has_attribute(__fallthrough__)`?

Reply:
Sure.  I'll update this PR tomorrow.

Reply:
@nicowilliams ping :)

Reply:
```C
#if defined(DISPATCH_FALLTHROUGH)
# define JQ_FALLTHROUGH DISPATCH_FALLTHROUGH
#else
# if defined __has_attribute
#  if __has_attribute(__fallthrough__)
#   define JQ_FALLTHROUGH __attribute__((fallthrough))
#  endif
# else
#  define JQ_FALLTHROUGH do {} while (0) /* fallthrough */
# endif
#endif
```
look ok?